src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/DecompressionHandler.cs

```diff
@@ -395,7 +395,7 @@ public override ValueTask<int> ReadAsync(Memory<byte> buffer, CancellationToken
                         }
 
                         Debug.Assert(_firstByteStatus == FirstByteStatus.Consumed);
-                        return _stream.ReadAsync(buffer, cancellationToken);
+                        return _stream.ReadAsync(buffer, cancellationToken).ConfigureAwait(false);
```

Comment:
This is returning, not awaiting. ConfigureAwait is not useful here, and this won't compile.

Reply:
right ! fixed